validation/pidfile.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	tap "github.com/mndrix/tap-go"
+	"github.com/opencontainers/runtime-tools/validation/util"
+)
+
+func main() {
+	t := tap.New()
+	t.Header(0)
+
+	tempDir, err := ioutil.TempDir("", "oci-pid")
+	if err != nil {
+		util.Fatal(err)
+	}
+	tempPidFile := filepath.Join(tempDir, "pidfile")
+	defer os.RemoveAll(tempDir)
```

Comment:
nit: I'd rather swap the order of these two lines to get the cleanup `defer` right after the `TempDir` error check.  `filepath.Join` cannot error, but having the `defer` set as early as possible would give me warm fuzzy feelings ;).

Reply:
OK, I'll update it.